extensions/src/main/scala/ai/h2o/sparkling/extensions/rest/api/ImportFrameHandler.scala

```diff
@@ -63,6 +63,9 @@ class ImportFrameHandler extends Handler {
 
     convertCategoricalColumnsToOtherTypesIfNeeded(frame, categoricalColumnIndices)
 
+    // Final frame created -> log the same information H2O would log after import is finished
+    ParseDataset.logParseResults(frame);
```

Comment:
We need to add a null pointer handling. Error from on of SW tests:
```
Server error: {"__meta":{"schema_version":3,"schema_name":"H2OErrorV3","schema_type":"H2OError"},"timestamp":1618932603367,"error_url":"/3/FinalizeFrame","msg":"\n\nERROR MESSAGE:\n\nCaught exception: 
java.lang.NullPointerException\n\n","dev_msg":"\n\nERROR MESSAGE:\n\nCaught exception: 
java.lang.NullPointerException from: 
water.parser.ParseDataset.logParseResults(ParseDataset.java:1059)\n\n","http_status":500,"values":
{},"exception_type":"java.lang.NullPointerException","exception_msg":"\n\nERROR 
MESSAGE:\n\nnull\n\n","stacktrace":["java.lang.NullPointerException","    
water.parser.ParseDataset.logParseResults(ParseDataset.java:1059)","    
ai.h2o.sparkling.extensions.rest.api.ImportFrameHandler.finalize(ImportFrameHandler.scala:67)","    
sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)","    
sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)","    
sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)","    
java.lang.reflect.Method.invoke(Method.java:498)","   
water.api.Handler.handle(Handler.java:60)","    
water.api.RequestServer.serve(RequestServer.java:470)","    
water.api.RequestServer.doGeneric(RequestServer.java:301)","    
water.api.RequestServer.doPost(RequestServer.java:227)","    
javax.servlet.http.HttpServlet.service(HttpServlet.java:727)","    
javax.servlet.http.HttpServlet.service(HttpServlet.java:820)","    
org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:565)","    
org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:479)","    
org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1031)","    
org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:406)","    
org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:965)","    
org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:117)","    
org.eclipse.jetty.server.handler.HandlerCollection.handle(HandlerCollection.java:149)","    
org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:111)","    
water.webserver.jetty8.Jetty8ServerAdapter$LoginHandler.handle(Jetty8ServerAdapter.java:130)","    
org.eclipse.jetty.server.handler.HandlerCollection.handle(HandlerCollection.java:149)","    
org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:111)","    
org.eclipse.jetty.server.Server.handle(Server.java:349)","    
org.eclipse.jetty.server.AbstractHttpConnection.handleRequest(AbstractHttpConnection.java:449)","    
org.eclipse.jetty.server.AbstractHttpConnection$RequestHandler.content(AbstractHttpConnection.java:925)","    
org.eclipse.jetty.http.HttpParser.parseNext(HttpParser.java:857)","    
org.eclipse.jetty.http.HttpParser.parseAvailable(HttpParser.java:235)","    
org.eclipse.jetty.server.AsyncHttpConnection.handle(AsyncHttpConnection.java:76)","    
org.eclipse.jetty.io.nio.SelectChannelEndPoint.handle(SelectChannelEndPoint.java:609)","    
org.eclipse.jetty.io.nio.SelectChannelEndPoint$1.run(SelectChannelEndPoint.java:45)","    
org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:599)","    
org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:534)","   
java.lang.Thread.run(Thread.java:748)"]}
```

Reply:
That is very suspicious because it means the "first" vec of the Frame is actually missing. That shouldn't be the case at this point.